rx-preferences/src/main/java/com/f2prateek/rx/preferences2/RealPreference.java

```diff
@@ -80,7 +79,9 @@
   }
 
   @Override public void delete() {
```

Comment:
apply is synchronous in memory. But, that's not the problem.
`delete` could be called in between the `contains` and `adapter.get` call in `RealPreference.get`.